Translations/translation_RO.json

```diff
@@ -0,0 +1,214 @@
+      "text2": ["Interfată", "utilizator"],
+      "desc": "Setări interfată utilizator"
+    },
+    "AdvancedMenu": {
+      "text2": ["Optiuni", "avansate"],
+      "desc": "Optiuni avansate"
+    }
+  },
+  "menuOptions": {
+    "DCInCutoff": {
+      "text2": ["Sursa de", "alimentare"],
+      "desc": "Sursa de alimentare. Setează tensiunea de întrerupere. (DC 10V) (S 3.3V per celulă, dezactivati limita de alimentare)"
+    },
+    "SleepTemperature": {
+      "text2": ["Temp", "repaus"],
+      "desc": "Temperatura vârfului în \"modul repaus\""
+    },
+    "SleepTimeout": {
+      "text2": ["Expirare", "repaus"],
+      "desc": "Interval înainte de lansarea \"modului de repaus\" în (S=secunde | M=minute)"
+    },
+    "ShutdownTimeout": {
+      "text2": ["Expirare", "oprire"],
+      "desc": "Interval înainte ca letconul să se oprească (M=minute)"
+    },
+    "MotionSensitivity": {
+      "text2": ["Sensibilitate", "la mişcare"],
+      "desc": "0=oprit | 1=putin sensibil | ... | 9=cel mai sensibil"
+    },
+    "TemperatureUnit": {
+      "text2": ["Unitate de", "temperatură"],
+      "desc": "C=Celsius | F=Fahrenheit"
+    },
+    "AdvancedIdle": {
+      "text2": ["Detalii,", "ecran inactiv"],
+      "desc": "Afisati informatii detaliate într-un font mai mic pe ecranul de repaus"
+    },
+    "DisplayRotation": {
+      "text2": ["Orientare", "ecran"],
+      "desc": "R=dreptaci | L=stângaci | A=auto"
+    },
+    "BoostTemperature": {
+      "text2": ["Boost", "temp"],
+      "desc": "Temperatura utilizată în \"modul boost\""
+    },
+    "AutoStart": {
+      "text2": ["Auto", "start"],
+      "desc": "Start letcon în modul de lipire la pornire (O=oprit | S=lipire | Z=repaus | R=repaus la temperatura camerei)"
+    },
+    "CooldownBlink": {
+      "text2": ["Clipeste", "la răcire"],
+      "desc": "Clipeste temperatura după oprirea încălzirii, în timp ce vârful este încă fierbinte"
+    },
+    "TemperatureCalibration": {
+      "text2": ["Calibrare", "temperatură?"],
+      "desc": "Începeti calibrarea compensării de temperatură a vârfului"
+    },
+    "SettingsReset": {
+      "text2": ["Setări", "din fabrică"],
+      "desc": "Reveniti la setările din fabrică"
+    },
+    "VoltageCalibration": {
+      "text2": ["Calibrare tens.", "de intrare?"],
+      "desc": "Porniti calibrarea VIN (apăsati lung pentru a iesi)"
+    },
+    "AdvancedSoldering": {
+      "text2": ["Detalii", "ecran lipire"],
+      "desc": "Afisati informatii detaliate într-un font mai mic pe ecranul de lipire"
+    },
+    "ScrollingSpeed": {
+      "text2": ["Viteză", "derulare"],
+      "desc": "Viteză derulare text cu informatii la (S=lent | F=rapid)"
+    },
+    "QCMaxVoltage": {
+      "text2": ["QC", "voltaj"],
+      "desc": "Tensiunea maximă QC dorită pentru care negociază letconul"
+    },
+    "PDNegTimeout": {
+      "text2": ["PD", "timeout"],
+      "desc": "PD negotiation timeout in 100ms steps for compatibility with some QC chargers"
```

Comment:
@sandmanRO

Reply:
Timp limita de negociere pentru tranzactia PD, in pasi de 100ms, pentru compatibilitate cu alimentatoarele QC